README.md

```diff
@@ -8,6 +8,46 @@
+
+* [Compute Canada: Systems available](https://docs.alliancecan.ca/wiki/Getting_started#What_systems_are_available?) - A brief introduction for the five systems of Compute Canada and the clusters' types.
+* [Compute Canada: Allocations and compute scheduling](https://docs.alliancecan.ca/wiki/Allocations_and_compute_scheduling#Choosing_GPU_models_for_your_project) - A brief tutorial of choosing GPU models for your project.
+* [Compute Canada: Checkpoints](https://docs.alliancecan.ca/wiki/Points_de_contr%C3%B4le/en) - A tutorial of creating and loading a checkpoint.
+* [Compute Canada: Machine Learning tutorial](https://docs.alliancecan.ca/wiki/Tutoriel_Apprentissage_machine/en#Checkpointing_a_long-running_job) - A toturial of checkpointing a long-running job.
+* [Tensorflow: Checkpoint guide](https://www.tensorflow.org/guide/checkpoint) - Guidance for training checkpoints while using Tensorflow.
+* [Enabling Coordinated Checkpointing for Distributed HPC Application](https://www.youtube.com/watch?v=lswkWQU4_W0) - A Youtube video presenting a new coordination mechanism for Checkpoint/Restore in User-Space.
+* [Nvidia: MIG user guide documentation](https://docs.nvidia.com/datacenter/tesla/mig-user-guide/index.html#abstract) - A user guide documentation of multi-instance GPU from Nvidia.
+* [Tensorflow: Distributed training](https://www.tensorflow.org/guide/keras/distributed_training) - Introduction and tutorial of Multi-GPU and distributed training.
+* [Wikipedia: GNU parallel](https://en.wikipedia.org/wiki/GNU_parallel) - Introduction and user's guide of GNU parallel.
+* [Compute Canada: GNU parallel](https://docs.alliancecan.ca/wiki/GNU_Parallel) - An introductory tutorial of using GNU parallel.
+* [GNU parallel tutorial documentation](https://www.gnu.org/software/parallel/parallel_tutorial.html) - A tutorial of GNU parallel, including its functionality, options and syntax.
+* [Open-MPI documentation](https://www.open-mpi.org/doc/) - A series of versions for Open MPI documentations.
+* [Open MPI v5.0.x](https://docs.open-mpi.org/en/v5.0.x/) - The documentation of current release Open MPI.
+* [SLURM: MPI users guide](https://slurm.schedmd.com/mpi_guide.html) - An introductory tutorial of various MPI implementations.
+* [Github: Open-MPI tutorial](https://usc-rc.github.io/tutorials/open-mpi) - A tutorial of using Open MPI.
+* [Compute Canada: MPI-IO](https://docs.alliancecan.ca/wiki/MPI-IO) - Description and using tutorial of MPI-IO.
+* [jax.pmap](https://jax.readthedocs.io/en/latest/_autosummary/jax.pmap.html) - Step-by-step jax.pmap guide.
+* [Compute Canada: Flax](https://docs.alliancecan.ca/wiki/Flax) - Introduction and tutorial of Flax, a neural network library and ecosystem for JAX that is designed for flexibility, including the guidance of using jax.pmap.
+* [Ashoka HPC Documentation](https://hpc.ashoka.edu.in/An%20Introduction%20to%20HPC/When%20should%20I%20use%20HPC/) - A brief description of when should we use HPC.
```

Comment:
It looks like this link doesn't actually contain much information. I think we should just remove it here and capture that information in the long-form internal document instead